test/integ/pandas/test_pandas_tools.py

```diff
@@ -45,10 +45,14 @@
 
 @pytest.mark.parametrize("quote_identifiers", [True, False])
 @pytest.mark.parametrize("auto_create_table", [True, False])
+@pytest.mark.parametrize("index", [True, False])
+@pytest.mark.parametrize("engine", ["auto", "pyarrow"])
```

Comment:
These tests are really slow relatively to the others. I'd like to not make it run 16 times instead of 4.
Do you think that it'd be enough to test `**kwargs` with sending in constant values in the already existing 4 tests, instead of the parametrized ones?